bazel/external/quiche.BUILD

```diff
@@ -86,6 +86,72 @@ envoy_cc_test_library(
     deps = [":http2_platform"],
 )
 
+envoy_cc_library(
+    name = "http2_core_http2_priority_write_scheduler_lib",
+    hdrs = ["quiche/http2/core/http2_priority_write_scheduler.h"],
+    copts = quiche_copts,
+    repository = "@envoy",
+    deps = [
+        ":http2_core_write_scheduler_lib",
+        ":spdy_core_intrusive_list_lib",
+        ":spdy_core_protocol_lib",
+        ":spdy_platform",
+    ],
+)
+
+envoy_cc_test_library(
+    name = "http2_core_http2_priority_write_scheduler_test",
+    srcs = [
+        "quiche/http2/core/http2_priority_write_scheduler_test.cc",
+    ],
+    copts = quiche_copts,
+    repository = "@envoy",
+    deps = [
+        ":http2_core_http2_priority_write_scheduler_lib",
+        ":quiche_common_platform_test_helpers_lib",
+        ":quiche_common_test_tools_test_utils_lib",
+        ":spdy_platform",
+    ],
+)
+
+envoy_cc_library(
+    name = "http2_core_priority_write_scheduler_lib",
```

Comment:
s/_lib/_interface

Reply:
Discussed offline: I removed the `_interface` convention from this file for consistency